src/libs/SidebarUtils.ts

```diff
@@ -437,7 +437,7 @@ function getOptionData(
     result.displayNamesWithTooltips = displayNamesWithTooltips;
 
     if (status) {
-        result.status = status;
+        result.status = status.text;
```

Comment:
Why was this changed?

Reply:
This will stop statuses from displaying in the LHN as we access properties of the JSON object in `OptionRowLHN`, e.g. [here](https://github.com/Expensify/App/blob/main/src/components/LHNOptionsList/OptionRowLHN.js#L151).